### PR TITLE
Add tooltips to items in the menu bar

### LINKED ISSUE #339: Entire feedback button should be clickable
Current behaviour: only icon is clickable

Screen Shot 2022-10-02 at 9 19 58 pm

Expected behaviour: the entire area around the icon and the text are a single click target

Reasoning: I went to click it to test it out and assumed that clicking the text would make the popup appear, but you actually have to click the icon itself - and there is also almost no padding, so you have to click it exactly. There is basically no downside to making the whole area clickable since there are no other click targets nearby to get confused with.

### PR DESCRIPTION
**Pull request by jhea0008**
## Summary

Improves the tooltips to be closer to the design and adds them to the video controller

<hr/>

### Why is this change needed?
aesthetics

<hr/>

### What did you change?
New componet: ProjectToolTip with styling in it

<hr/>

### Screenshots of UI changes, if any
Screen Shot 2022-10-14 at 12 51 53

<hr/>

### OS

- [ ] Linux
- [X] MacOS
- [ ] Windows

### CODE REVIEW

**Comment by jhea0008** [comment]
Addresses issue: 
https://github.com/MLVETDevelopers/mlvet/issues/339

File: src/renderer/pages/Project.tsx
Comment (chloebrett) [inline src/renderer/pages/Project.tsx]: This whole list is pretty repetitive, could we just have a list of objects containing title, action and icon and then render the buttons by mapping over the list? Or at least a helper function to generate a button given a tooltip title, action and icon

File: src/renderer/components/Editor/ProjectTooltip.tsx
Comment (chloebrett) [inline src/renderer/components/Editor/ProjectTooltip.tsx]: I think there's too much going on inline here, can we separate out into a couple of steps so it's clearer what's going on? Might be able to get a definitive type here as well if that's still an issue

File: src/renderer/pages/Project.tsx
Comment (jhea0008) [reply src/renderer/pages/Project.tsx]: Created a new component to encapsulate more of the features common to all the buttons

File: src/renderer/components/Editor/ProjectTooltip.tsx
Comment (jhea0008) [reply src/renderer/components/Editor/ProjectTooltip.tsx]: Yea, I just copy pasted this from the mui examples, so I'm not 100% clear on it either. I have separated it out a bit I think it's clearer now

File: src/renderer/components/Editor/ProjectTooltip.tsx
Comment (Mhudlark) [inline src/renderer/components/Editor/ProjectTooltip.tsx]: Can it just be `{props.children}`? Is the wrapping <span> needed?

File: src/renderer/components/Editor/MenuBarButton.tsx
Comment (Mhudlark) [inline src/renderer/components/Editor/MenuBarButton.tsx]: Can it just be `{props.children}`? Is the wrapping <div> needed?

**Review by Mhudlark — APPROVED** [review]
Nice work!
Approving because it works well when tested and little time. Few small comments regarding code implementation, would be nice if they were fixed before merging.

**Review by emmapainter — APPROVED** [review]
Thanks for smashing this out Justin, this will do the job just fine 😄

**Pull request merged**
